lib/plugins/wallet/monerod/monerod.js

```diff
@@ -235,12 +235,21 @@ function checkBlockchainStatus (cryptoCode) {
   })
 }
 
+function getTxHashesByAddress (cryptoCode, address) {
+  checkCryptoCode(cryptoCode)
+    .then(() => refreshWallet())
+    .then(() => fetch('get_address_index', { address: address }))
+    .then(addressRes => fetch('get_transfers', { in: true, pool: true, pending: true, account_index: addressRes.index.major, address_indices: [addressRes.index.minor] }))
```

Comment:
One of the parameters is wrong, according to https://www.getmonero.org/resources/developer-guides/wallet-rpc.html#get_transfers

```suggestion
    .then(addressRes => fetch('get_transfers', { in: true, pool: true, pending: true, account_index: addressRes.index.major, subaddr_indices: [addressRes.index.minor] }))
```

Reply:
Good catch!